cognite/client/_api/simulators/simulation_runs.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+from cognite.client._api_client import APIClient
+from cognite.client._constants import DEFAULT_LIMIT_READ
+from cognite.client.data_classes.simulators.filters import SimulationRunsFilter
+from cognite.client.data_classes.simulators.simulators import SimulationRun, SimulationRunsList
+from cognite.client.utils._experimental import FeaturePreviewWarning
+
+if TYPE_CHECKING:
+    from cognite.client import ClientConfig, CogniteClient
+
+
+class SimulatorRunsAPI(APIClient):
```

Comment:
Shouldn't we include the method to fetch [simulation data](https://api-docs.cognite.com/20230101/tag/Simulation-Runs/operation/simulation_data_by_run_id_simulators_runs_data_list_post) here too? We can always add it in a future PR, but if this is inteded to cover all read endpoints, this one is missing.

Reply:
To be added in a follow up PR